src/driver/main.cpp

```diff
@@ -820,22 +824,23 @@ int main(int argc, const char* argv[])
     }
 
     auto&& m = get_commands();
-    auto cmd = args.front();
+    auto cmd_migx = args.front();
 
-    if(cmd == "ort-sha")
+    if(cmd_migx == "ort-sha")
     {
         std::cout << MIGRAPHX_ORT_SHA1 << std::endl;
         return 0;
     }
 
-    if(m.count(cmd) > 0)
+    if(m.count(cmd_migx) > 0)
     {
-        m.at(cmd)(argv[0], {args.begin() + 1, args.end()});
+        m.at(cmd_migx)(argv[0], {args.begin() + 1, args.end()});
     }
     else
     {
         run_command<main_command>(argv[0], args);
     }
 
+    std::cout << "command [ " << cmd_string << " ] was successful." << std::endl;
```

Comment:
```suggestion
    std::cout << "command: [ " << cmd_string << " ] was successful." << std::endl;
```

Reply:
`was successful` part does it get print even if it runs `main_command` ? I think we can omit printing `was successful`.

Reply:
I've fixed the logic for this. It won't print for `main_command` anymore.